### DISCUSSION TITLE
High CPU Usage and Power Consumption After Updating to Dify 0.11.1

### DISCUSSION THREAD

**Discussion by Hanxiao-Adam-Qi** [langgenius/dify#10664] [Help] [answered]
### Self Checks

- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Hi folks,

After updating Dify to version 0.11.1, I noticed that my server’s power consumption increased significantly from **12W** to **50W**. I checked the CPU usage with docker stats and found the following information:

```
CONTAINER ID   NAME                             CPU %     MEM USAGE / LIMIT     MEM %     NET I/O           BLOCK I/O        PIDS
68a5f7fb5fc2   docker-nginx-1                   0.00%     10.76MiB / 15.46GiB   0.07%     8.64kB / 0B       500kB / 16.4kB   14
673ecc1f4126   docker-api-1                     0.02%     758.3MiB / 15.46GiB   4.79%     65.6kB / 11.5kB   26.8MB / 126MB   11
3eef20a570cd   docker-web-1                     0.13%     61.43MiB / 15.46GiB   0.39%     13.2kB / 1.65kB   0B / 28.7kB      34
3b1a019f1355   docker-db-1                      0.92%     18.98MiB / 15.46GiB   0.12%     22.2kB / 13.4kB   205kB / 414kB    6
ddf8393e2995   docker-ssrf_proxy-1              0.01%     12.99MiB / 15.46GiB   0.08%     18.3kB / 1.37kB   0B / 885kB       9
c42ed25b6703   docker-sandbox-1                 0.00%     36.58MiB / 15.46GiB   0.23%     10.2kB / 1.12kB   8.19kB / 178MB   13
1b6e0218865d   docker-redis-1                   0.07%     2.621MiB / 15.46GiB   0.02%     15.5kB / 4.05kB   131kB / 0B       5
fa45849745e9   docker-weaviate-1                0.07%     111.2MiB / 15.46GiB   0.70%     9.43kB / 0B       254kB / 0B       12
fd7467e2c6ca   firecrawl-worker-1               0.00%     70.52MiB / 15.46GiB   0.45%     79.7kB / 145kB    9.52MB / 0B      13
051dbb043aca   firecrawl-api-1                  0.01%     66.03MiB / 15.46GiB   0.42%     38.7kB / 2.28kB   13.1MB / 0B      13
7b671836343a   firecrawl-playwright-service-1   0.00%     211.3MiB / 15.46GiB   1.33%     9.97kB / 0B       156MB / 442kB    62
257f8ad751a4   firecrawl-redis-1                0.37%     7.84MiB / 15.46GiB    0.05%     157kB / 99.2kB    4.15MB / 0B      6
beccfd17151d   docker-worker-1                  226.46%   116.7MiB / 15.46GiB   0.74%     2.26kB / 0B       0B / 0B          13
```

The CPU usage is always higher than 100% for ```docker-worker-1```. Not sure if this is normal nor something. I have re-copied the ```.env.example``` file to ```.env```, but nothing changed.

Thanks in advance for any help!

Other information:
_System_: Ubuntu 24.04.1 LTS
_Deploy method:_ Self-hosted, docker
_version:_ v0.11.1

**Comment by kurokobo** [comment, marked as answer]
https://github.com/langgenius/dify/issues/10619

**Comment by loganclark360** [comment]
> ### Self Checks
> 
> - [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
> - [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
> - [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
> - [X] Please do not modify this template :) and fill in all the required fields.
> 
> ### Content
> 
> Hi folks,
> 
> 
> 
> After updating Dify to version 0.11.1, I noticed that my server’s power consumption increased significantly from **12W** to **50W**. I checked the CPU usage with docker stats and found the following information:
> 
> 
> 
> ```
> 
> CONTAINER ID   NAME                             CPU %     MEM USAGE / LIMIT     MEM %     NET I/O           BLOCK I/O        PIDS
> 
> 68a5f7fb5fc2   docker-nginx-1                   0.00%     10.76MiB / 15.46GiB   0.07%     8.64kB / 0B       500kB / 16.4kB   14
> 
> 673ecc1f4126   docker-api-1                     0.02%     758.3MiB / 15.46GiB   4.79%     65.6kB / 11.5kB   26.8MB / 126MB   11
> 
> 3eef20a570cd   docker-web-1                     0.13%     61.43MiB / 15.46GiB   0.39%     13.2kB / 1.65kB   0B / 28.7kB      34
> 
> 3b1a019f1355   docker-db-1                      0.92%     18.98MiB / 15.46GiB   0.12%     22.2kB / 13.4kB   205kB / 414kB    6
> 
> ddf8393e2995   docker-ssrf_proxy-1              0.01%     12.99MiB / 15.46GiB   0.08%     18.3kB / 1.37kB   0B / 885kB       9
> 
> c42ed25b6703   docker-sandbox-1                 0.00%     36.58MiB / 15.46GiB   0.23%     10.2kB / 1.12kB   8.19kB / 178MB   13
> 
> 1b6e0218865d   docker-redis-1                   0.07%     2.621MiB / 15.46GiB   0.02%     15.5kB / 4.05kB   131kB / 0B       5
> 
> fa45849745e9   docker-weaviate-1                0.07%     111.2MiB / 15.46GiB   0.70%     9.43kB / 0B       254kB / 0B       12
> 
> fd7467e2c6ca   firecrawl-worker-1               0.00%     70.52MiB / 15.46GiB   0.45%     79.7kB / 145kB    9.52MB / 0B      13
> 
> 051dbb043aca   firecrawl-api-1                  0.01%     66.03MiB / 15.46GiB   0.42%     38.7kB / 2.28kB   13.1MB / 0B      13
> 
> 7b671836343a   firecrawl-playwright-service-1   0.00%     211.3MiB / 15.46GiB   1.33%     9.97kB / 0B       156MB / 442kB    62
> 
> 257f8ad751a4   firecrawl-redis-1                0.37%     7.84MiB / 15.46GiB    0.05%     157kB / 99.2kB    4.15MB / 0B      6
> 
> beccfd17151d   docker-worker-1                  226.46%   116.7MiB / 15.46GiB   0.74%     2.26kB / 0B       0B / 0B          13
> 
> ```
> 
> 
> 
> The CPU usage is always higher than 100% for ```docker-worker-1```. Not sure if this is normal nor something. I have re-copied the ```.env.example``` file to ```.env```, but nothing changed.
> 
> 
> 
> Thanks in advance for any help!
> 
> 
> 
> Other information:
> 
> _System_: Ubuntu 24.04.1 LTS
> 
> _Deploy method:_ Self-hosted, docker
> 
> _version:_ v0.11.1
